README.rst

```diff
@@ -295,8 +295,17 @@ Placeholder          Definition
                      is the name of the branch to which the push was made (or
                      possibly the tag that was pushed, if using Appveyor); for
                      ``release``, this is the name of the tag
-``{commit}``         The hash of the commit the build ran against or that was
-                     tagged for the release
+``{build_commit}``   The hash of the commit the build ran against or that was
+                     tagged for the release.  Note that, for PR builds on
+                     Travis and Appveyor, this is the hash of an autogenerated
+                     merge commit.
+``{commit}``         The hash of the original commit that triggered the build
+                     or that was tagged for the release.  For GitHub Actions
+                     and all non-pull request builds, this is always the same
+                     as ``{build_commit}``.  For pull request builds on
+                     Appveyor, this is the head of the PR branch.  For pull
+                     request builds on Travis, this is ``UNK``, as the API does
+                     not expose the original commit.
```

Comment:
but I still hope we workaround as discussed in https://github.com/con/tinuous/issues/55#issuecomment-849949409, right?

Reply:
If you mean pulling PR branches from GitHub, I consider that overkill.

Reply:
but we do not have a better solution, do we?
The only other workaround I see is particular to DataLad setup since would require Appveyor run (now that we know that merge is done by github so the same across CIs) is to take information from Appveyor run which knows the `commit` for the `build_commit` and use for all CIs.  Might also be needed/desired to establish persistent cache of knowing such mappings from `build_commit` to `commit`.

Reply:
The best alternative I can think of is querying the GitHub API for information on the PR, checking whether the merge commit is the same as the build commit, and, if so, using the head commit listed there.  It'll require some rewriting of the code that I'd prefer to do in a separate PR.

Reply:
ok, thank you, let's proceed then